src/axom/slic/tests/slic_macros.cpp

```diff
@@ -412,6 +413,67 @@ TEST(slic_macros, test_tagged_macros)
   EXPECT_TRUE(slic::internal::is_stream_empty());
 }
 
+//------------------------------------------------------------------------------
+TEST(slic_macros, test_macros_file_output)
+{
+  std::string msgfmt = "<MESSAGE>";
+
+  // GenericOutputStream(std::string stream) and
+  // GenericOutputStream(std::string stream, std::string format) constructors
+  // do not create a a file until macros called, then flushed
+  std::string no_fmt = "file_no_fmt.txt";
+  std::string with_fmt = "file_with_fmt.txt";
+
+  slic::addStreamToAllMsgLevels(new slic::GenericOutputStream(no_fmt));
+
+  slic::addStreamToAllMsgLevels(new slic::GenericOutputStream(with_fmt, msgfmt));
+
+  EXPECT_EQ(axom::utilities::filesystem::pathExists(no_fmt), false);
+  EXPECT_EQ(axom::utilities::filesystem::pathExists(with_fmt), false);
```

Comment:
Minor: These might be slightly easier to read using `EXPECT_TRUE()` and `EXPECT_FALSE()` 
instead of `EXPECT_EQ(, {true,false}`